13/main.js

```diff
@@ -0,0 +1,79 @@
+const field = document.getElementById("field");
+const buttonFromTask1 = document.getElementById("button1");
```

Comment:
Try to give more meaningful names for variables, timerButton/timeoutButton looks much better